test/integration/src/py/test_liquidity_pools.py

```diff
@@ -250,6 +286,81 @@ def test_add_faucet_coins(
+    sifchain_symbol = ("c" + new_currency["newtoken_symbol"]).lower()
+
+    logging.info(f"request: {basic_transfer_request}")
+    basic_transfer_request.ethereum_address = source_ethereum_address
+    basic_transfer_request.check_wait_blocks = True
+    target_rowan_balance = 10 ** 18
+    request, credentials = generate_test_account(
+        basic_transfer_request,
+        rowan_source_integrationtest_env_transfer_request,
+        rowan_source_integrationtest_env_credentials,
+        target_ceth_balance=100,
+        target_rowan_balance=target_rowan_balance
+    )
+    #Transfer Newly Generated Currency
+    transfer_new_currency(request, sifchain_symbol, new_currency, target_new_currency_balance)
+    time.sleep(10)
+
+    sifaddress = request.sifchain_address
+    logging.info("get balances just to have those commands in the history")
+    balance = test_utilities.get_sifchain_addr_balance(sifaddress, basic_transfer_request.sifnodecli_node, "rowan")
+    balance_ceth = test_utilities.get_sifchain_addr_balance(sifaddress, basic_transfer_request.sifnodecli_node, "ceth")
+    balance_new_currency = test_utilities.get_sifchain_addr_balance(sifaddress, request.sifnodecli_node, sifchain_symbol)
+    
+    logging.info(sifchain_symbol+" balance="+ str(balance_new_currency))
+    logging.info("ceth  balance="+ str(balance_ceth))
+    logging.info("rowan  balance="+ str(balance))
+
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    basic_transfer_request.amount = 50000
+    basic_transfer_request.sifchain_symbol = "rowan"
+    basic_transfer_request.sifchain_address = sifaddress
+    add_faucet_coins(basic_transfer_request, credentials)
+    time.sleep(10)
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    balance = test_utilities.get_sifchain_addr_balance(sifaddress, basic_transfer_request.sifnodecli_node, "rowan")
+
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    basic_transfer_request.amount = 10
+    basic_transfer_request.sifchain_symbol = "ceth"
+    basic_transfer_request.sifchain_address = sifaddress
+    add_faucet_coins(basic_transfer_request, credentials)
+    time.sleep(10)
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    balance_ceth = test_utilities.get_sifchain_addr_balance(sifaddress, basic_transfer_request.sifnodecli_node, "ceth")
+
+
+    get_faucet_balance(request.sifnodecli_node)
+    request.amount = 200000
+    request.sifchain_symbol = sifchain_symbol
+    request.sifchain_address = sifaddress
+    add_faucet_coins(request, credentials)
+    time.sleep(10)
+    get_faucet_balance(request.sifnodecli_node)
+    balance_new_currency = test_utilities.get_sifchain_addr_balance(sifaddress, request.sifnodecli_node, sifchain_symbol)
+  
+
+    #get faucet balances
+    logging.info(sifchain_symbol+" balance="+ str(balance_new_currency))
+    logging.info("ceth  balance="+ str(balance_ceth))
```

Comment:
Using formatted logging is stylistically nicer.  Also, some python tools complain about white imbalanced +, so that would avoid that issue too.

Reply:
So like:
`logging.info("ceth balance {balance_ceth}")`
for example.

Reply:
done